.pre-commit-config.yaml

```diff
@@ -8,6 +8,8 @@ repos:
     rev: 5.10.1
     hooks:
       - id: isort
+        args: [--add-import, from __future__ import annotations]
+        exclude: ^tests/ui/(helpers|test_exception_trace).py$
```

Comment:
Excluding these for now since the test checks exact line numbers and thus causes a lot of churn when those change: https://github.com/python-poetry/cleo/blob/28eb7bad5762f9b6a0cb5c0cbd0b11efc92fc8ca/tests/ui/test_exception_trace.py#L31-L33